CONTRIBUTING.rst

```diff
@@ -664,6 +664,63 @@ Later on
    above depending if you have more commits that cause conflicts in your PR (rebasing applies each
    commit from your PR one-by-one).
 
+How to communicate
+==================
+
+Apache Airflow is a Community within Apache Software Foundation. As the motto of
+the Apache Software Foundation states "Community over Code" - people in the
+community are far more important than the code they create.
+
+This means that communication is very important, and this chapter is all about it.
+
+We have various channel of communication - starting from the official devlist, comments
+in the Pull Requests, Slack, wiki.
+
+All those channels can be used for different purposes:
+
+* The devlist for official communication, discussing proposals, voting, general issues, asking community for
+  opinion etc.
+* Wiki for detailed discussions on big proposals (AIPs) and some helpful, shared resources.
+* PRs for discussing implementation details of this PR (not for architectural discussions)
+* Slack for ad-hoc questions, troubleshooting, occassional discussions, group talks (including SIG - special
+interest groups), notifications, random queries, newbie questions etc.
+
+
+It is important to remember that all contributors, committers and even PMC members are individuals
+that contribute a lot of their free time on top of their work time to Apache Airflow.
+They have families, personal life, sometimes they are swamped with work, sometimes they are on vacations
```

Comment:
Maybe we should change "they" to "we" to create a stronger bond and avoid deindividuation? Especially that "we" is used later on.

Reply:
Changed to "commiters" and reworded we in a few places.